src/pages/lib/auth/manage-session/q/platform/[platform].mdx

```diff
@@ -0,0 +1,204 @@
+export const meta = {
+  title: `Manage user session and credentials`,
+  description: `Learn how to manage user sessions and credentials.`,
+};
+
+<InlineFilter filters={["js"]}>
```

Comment:
seems to me like other guides have react native as well except this one - I assume that is intentional?

Reply:
No that is a good call out - I just checked today and we can totally add react-native to this one - I will do that in my updates!